style.css

```diff
@@ -1,7 +1,13 @@
-@media (min-width: 375px), (min-width: 768px) {
+@media (min-width: 375px) {
```

Comment:
- [ ] you did a good jo implementing the desktop version good job so far 👍 but there is something you are missing out . kindly consider going back to your mobile version and fixing it all cause it was messed up when developing the desktop version remember we have to have a desktop version after the mobile version is okay and it has to stay okay to meet the responsiveness requirements while doing so consider also removing the horizontal scroll  in your mobile version design  doing all the above will help you to meet the below requirement 
<img width="1107" alt="Screen Shot 2023-06-09 at 09 24 38" src="https://github.com/Omwomo/Microverse-web-portfolio-project/assets/60146030/3420759f-3c51-40d3-a050-5d18c5f55ec9">